### PR TITLE
add bundle collapser to the generator

### PR DESCRIPTION
**Pull request by BrandonOCasey**

### CODE REVIEW

File: src/app/package-json.js [hunk 1]
```diff
@@ -143,8 +145,13 @@ const packageJSON = (current, context) => {
...
       ]),
 
+      'build:js:collapse': scriptify([
+        'bundle-collapser dist/%s.js',
+        '-o dist/%s.min.js'
+      ]),
+
       'build:js:uglify': scriptify([
-        'uglifyjs dist/%s.js',
+        'uglifyjs dist/%s.min.js',
```
Comment (misteroneill) [inline src/app/package-json.js]: Should this suffix be changed to `coll` or something instead of `min`? This confused me momentarily.

Comment (BrandonOCasey) [inline src/app/package-json.js]: well I am just treating bundle collapsing as a minification step

Comment (misteroneill) [inline src/app/package-json.js]: Ok, cool, that's fine. What about overwriting `dist/%s.js`?

**Comment by misteroneill** [comment]
LGTM

**Pull request merged**